pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -179,6 +179,13 @@ type ScrapeConfigSpec struct {
 	// +kubebuilder:validation:Enum=HTTP;HTTPS
 	// +optional
 	Scheme *string `json:"scheme,omitempty"`
+	// Indicator whether to request compressed response from the target.
```

Comment:
"borrowing" from the Prometheus docs: 
```suggestion
	// When false, Prometheus will request uncompressed response from the scraped target.
```

Reply:
Done. Change it